src/Writer/MSSQL.php

```diff
@@ -158,8 +168,38 @@ public function bcpCast(
         bool $nullable,
         int $version
     ): string {
-        $castFunction = $version > 10 ? 'TRY_CAST' : 'CAST';
+        // Binary types
+        if (in_array($type, self::$binaryTypes, true)) {
+            $convertFunction = $version > 10 ? 'TRY_CONVERT' : 'CONVERT';
+
+            // If image, we have to convert data to varbinary
+            if ($type === 'image') {
+                $type = 'varbinary';
+                $size = '(max)';
+            }
 
+            // Last parameter of the convert function is style:
+            // 0 - converted as string
+            // 1 - converted as hex value (if starts with 0x and length is even number
+            // Note: MsSQL cannot convert hex values with odd length
+            $style = sprintf(
```

Comment:
Ak je to validna HEX hodnota, tak pouzijeme `style = 1` a MsSQL hodnotu parsne, ... musi mat parny pocet znakov, tj. `0xaa` vie MsSQL spracovat, ale `0xaaa` nevie.